Code/Framework/AzCore/Platform/iOS/AzCore/NativeUI/NativeUISystemComponent_iOS.mm

```diff
@@ -58,14 +58,18 @@ static void OnDeadlock(NSString* nsTitle, NSString* nsMessage)
             freopen([pathForLog cStringUsingEncoding:NSASCIIStringEncoding], "a+", stderr);
             
             // These messages go to the file.
-            NSLog(@"BlockingDialog failed to display at %@", dateString);
+            NSLog(@"BlockingDialog might be failed to display at %@", dateString);
             
             NSLog(@"Title: %@", nsTitle);
             NSLog(@"Message: %@\n", nsMessage);
             
+            // There is no perfect solution because we do not know how the blocker thread handles the issue after the asserton
+            // abort() is commented out because we assume the asserted thread handles the issue well so the main thread is unblocked later.
+            // If there is a crash in the asserted thread then the app just crashes. From user perspective it is  similar to abort call.
+            /*
             NSLog(@"App terminated");
-            
             abort();
+            */
```

Comment:
Let's just remove this.  No need to keep the abort code around, even as a reference.  The comment block ("There is no perfect.. ") should be a function comment to explain what it does and why.

Reply:
Addressed.